locales/en.default.schema.json

```
@@ -51,9 +51,31 @@
             "label": "Standard"
           },
           "options__2": {
+            "label": "Arch"
+          },
+          "options__3": {
+            "label": "Blobs"
+          },
+          "options__4": {
             "label": "Card"
           },
-          "label": "Style"
+          "options__5": {
+            "label": "Chevron left"
+          },
+          "options__6": {
+            "label": "Chevron right"
+          },
+          "options__7": {
+            "label": "Circle"
+          },
+          "options__8": {
+            "label": "Diamond"
+          },
+          "options__9": {
+            "label": "Parallelogram"
+          },
+          "label": "Style",
+          "info": "Border and Shadow are only visible when Standard or Card styles are selected."
```

```suggestion
          "info": "Border and shadow are visible when style is set to Standard or Card."
```
Adding a suggestion to use more a positive form, to avoid using `only` and align with similar helper text. I know this might evolve depending of what we do for improving the clarity between the style of the whole card versus the style of the image only.